src/web/wfs/src/main/java/org/geoserver/wfs/web/WFSAdminPage.html

```diff
@@ -136,6 +136,10 @@
           <input wicket:id="disableStoredQueriesManagement" type="checkbox"></input>
           <label for="disableStoredQueriesManagement"><wicket:message key="disableStoredQueriesManagement">Disable stored queries management</wicket:message></label>
         </li>
+        <li>
```

Comment:
Since it's just a bugfix, it does not need a dedicated configuration to enable it...